CLI/ast.go

```diff
@@ -420,7 +420,14 @@ func (n *selectObjectNode) execute() (interface{}, error) {
 			return nil, err
 		}
 	}
-	return cmd.SetClipBoard(selection)
+	_, err = cmd.SetClipBoard(selection)
```

Comment:
The return of SetClipBoard is not useful because it sets State.Clipboard (which is the important thing), is that it?

Reply:
Yeah, I don't know why it is returning something in the first place